revolv/static/css/screen.css

```diff
@@ -10677,7 +11056,122 @@ header.edit-project-header > .container > h1 {
   #modal-how-it-works .info-row .video-area .btn-play:hover .icons, #modal-how-it-works .info-row .video-area .btn-play:focus .icons {
     background-position: -48px -44px;
   }
+
+
+/* 30-Jun */
+
+.price-tables .right-aside .payment-module .our_apps {
+  font-size: 13px;
+  margin-left: 0;
+  margin-right: 0;
+}
+.price-tables .right_asides aside.right-aside p.dummy {
+  color: #808080;
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 27px;
+  margin: 15px auto;
+  min-height: 197px;
+  text-align: center;
+  width: 100%;
+}
+.right_asides aside.right-aside .btn-i-want-to-donate {
+  width: 100%;
+}
+.details-active-project-module .title-white-border {
+  border: 2px solid #14b1e7;
+  font-size: 18px;
+  height: auto;
+  line-height: 42px;
+  padding: 0 45px;
 }
+header.header.after-scroll-header .rights {
+  padding: 38px 24px 0 0;
+}
+header.header {
+  height: 95px;
+  margin-top: 10px;
+}
+
+.footer .bottoms ul.social-ul li .icon-g-plus.icons {
+  background-position: -49px -8px;
+}
+.footer .bottoms ul.social-ul li:hover .icon-g-plus.icons {
+  background-position: -61px -8px;
+}
+
+
+
+
+
+
+
+}
+
```

Comment:
Same thing here: only one blank line and a comment about the fact that this is the section for mobile phone styles.